operations/power_management/Prepare_the_System_for_Power_Off.md

```diff
@@ -8,7 +8,7 @@ The `sat bootsys shutdown` and `sat bootsys boot` commands are used to shut down
 
 ### Prerequisites
 
-An authentication token is required to access the API gateway and to use the `sat` command. See the [System Security and Authentication](../security_and_authentication/System_Security_and_Authentication.md) and "SAT Authentication" in the System Admin Toolkit (SAT) product stream documentation.
+An authentication token is required to access the API gateway and to use the `sat` command. See the [System Security and Authentication](../security_and_authentication/System_Security_and_Authentication.md) section of this guide for information on how authentication tokens are used, and the "SAT Authentication" section of the HPE Cray EX System Admin Toolkit (SAT) product stream documentation (S-8031) for instructions on how to acquire a SAT authentication token.
```

Comment:
The page that the "System Security and Authentication" link here goes to was changed some time ago and no longer has the information that you are looking for. @nrockershousen may have a perspective on what the link here should be.

Reply:
My initial impression was that the `System Security and Authentication` page was linked in order to justify why the authentication step was necessary (i.e. the token is needed for API requests SAT makes.) Thus that plus the SAT product documentation combined would be sufficient to explain why and how to acquire a JWT for API calls. Still, @nrockershousen I'm also curious if there's a better page to link to here, please let me know.

Reply:
The actual procedure to retrieve an authentication token is: **operations/security_and_authentication/Retrieve_an_Authentication_Token.md**

The current link in the PR is just an overview of security and authentication in general. If you don't want to link to the actual procedure and want a list of all topics useful for authentication, I recommend linking to the security and authentication section the in operations index file:
**operations/index.md#security-and-authentication**

Reply:
I'm a little unsure about linking to the page on retrieving an authentication token by hand, as it doesn't specify how to use the retrieved token with SAT (and, as an aside, nor should it.) SAT has its own authentication mechanism through `sat auth` which essentially automates the documented authentication flow for SAT.

Linking to the Security and Authentication section index might be more appropriate in that case, though it may be confusing to the user as most of the subsections are not really relevant to retrieving an auth token for SAT.

My current leaning given the above is that it might be best to just remove the link entirely, and direct the user to the SAT product stream documentation exclusively.